doc/manual/src/language/derivations.md

```diff
@@ -8,98 +8,205 @@ It outputs an attribute set, and produces a [store derivation] as a side effect
 
-  Symbolic outputs of the derivation.
-  Each output name is passed to the [`builder`](#attr-builder) executable as an environment variable with its value set to the corresponding [output path].
+  Default: `[ "out" ]`
 
-  [output path]: @docroot@/glossary.md#gloss-output-path
+  Symbolic outputs of the derivation.
+  Each output name is passed to the [`builder`](#attr-builder) executable as an environment variable with its value set to the corresponding [store path].
 
-  By default, a derivation produces a single output path called `out`.
-  However, derivations can produce multiple output paths.
+  By default, a derivation produces a single output called `out`.
+  However, derivations can produce multiple outputs.
   This allows the associated [store objects](@docroot@/glossary.md#gloss-store-object) and their [closures](@docroot@/glossary.md#gloss-closure) to be copied or garbage-collected separately.
 
-  Examples:
-
-  Imagine a library package that provides a dynamic library, header files, and documentation.
-  A program that links against such a library doesn’t need the header files and documentation at runtime, and it doesn’t need the documentation at build time.
-  Thus, the library package could specify:
-
-  ```nix
-  derivation {
-    # ...
-    outputs = [ "lib" "dev" "doc" ];
-    # ...
-  }
-  ```
-
-  This will cause Nix to pass environment variables `lib`, `dev`, and `doc` to the builder containing the intended store paths of each output.
-  The builder would typically do something like
-
-  ```bash
-  ./configure \
-    --libdir=$lib/lib \
-    --includedir=$dev/include \
-    --docdir=$doc/share/doc
-  ```
-
-  for an Autoconf-style package.
-
-  You can refer to each output of a derivation by selecting it as an attribute, e.g. `myPackage.lib` or `myPackage.doc`.
-
-  The first element of `outputs` determines the *default output*.
-  Therefore, in the given example, `myPackage` is equivalent to `myPackage.lib`.
+  > **Example**
+  >
+  > Imagine a library package that provides a dynamic library, header files, and documentation.
+  > A program that links against such a library doesn’t need the header files and documentation at runtime, and it doesn’t need the documentation at build time.
+  > Thus, the library package could specify:
+  >
+  > ```nix
+  > derivation {
+  >   # ...
+  >   outputs = [ "lib" "dev" "doc" ];
+  >   # ...
+  > }
+  > ```
+  >
+  > This will cause Nix to pass environment variables `lib`, `dev`, and `doc` to the builder containing the intended store paths of each output.
+  > The builder would typically do something like
+  >
+  > ```bash
+  > ./configure \
+  >   --libdir=$lib/lib \
+  >   --includedir=$dev/include \
+  >   --docdir=$doc/share/doc
+  > ```
+  >
+  > for an Autoconf-style package.
+
+  The symbolic name is appended to an output's store path, unless it is `out`.
```

Comment:
```suggestion
  The name of an output is combined with the name of the derivation to create the name part of the outputs store path, unless it is `out`, in which case just the name of the derivation is used.
```

Reply:
I think it's good to be very explicit. "Symbolic name" I don't think is so clear, and the really is notion of output name just here but with e.g. `/nix/store/adsfasdfasdfasdf-drv^output` derived path syntax. There is no special case for `out` with that, for example.